src/components/Navbar/Navbar.js

```diff
@@ -10,10 +21,32 @@ export default function Navbar() {
         </div>
         <div className="nav-items">
           <a className="black-text-color">
-            <span className="material-icons ml-1 aqua-color" title="theme">
+            <span
+              className="material-icons ml-1 aqua-color"
+              title="profile details"
+            >
               account_circle
             </span>
           </a>
+          <div>
+            {authState.isUserLoggedIn && localStorage.getItem("token") ? (
+              <button
+                type="button"
+                className="btn orange-bg login-button ml-3 font-weight-bold btn-size"
+                onClick={() => logOutHandler()}
```

Comment:
You can simply write it as
```
onClick={logOutHandler}
```

Reply:
it should happen on onclick, in this way it will happen on page load. isn't it?